<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPreferencesPageGeneral</class>
 <widget class="QWidget" name="WdgPreferencesPageGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lLanguage">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cbLanguage"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lLogfileMode">
     <property name="text">
      <string>Logfile Mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbLogfileMode">
     <item>
      <property name="text">
       <string>No Logging</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Detailed Debug</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lLogfileModeInfo">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
